client/lib/importer/test/api-interaction.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import Dispatcher from 'dispatcher';
+import nock from 'nock';
+import partial from 'lodash/function/partial';
+
+import { fetchState } from '../actions';
+import { actionTypes } from '../constants';
+import store from '../store';
+
+const fetchTestState = partial( fetchState, 0 );
+const hydratedState = () => store.get().getIn( [ 'api', 'isHydrated' ] );
+const resetStore = () => Dispatcher.handleViewAction( { type: actionTypes.RESET_STORE } );
+
+const queuePayload = payload =>
+	nock( 'https://public-api.wordpress.com:443' )
+		.get( '/rest/v1.1/sites/0/imports/' )
+		.replyWithFile( 200, `${ __dirname }/api-payloads/${ payload }.json` );
```

Comment:
thanks for the links @gwwar! I originally had these in one file as JavaScript objects but that was cumbersome to modify and add to (due to the syntax differences between JavaScript object literals and JSON objects), so I moved them into their own files to simplify the process. the payloads aren't particularly complicated, but some of the ones that will come with further testing would definitely be noisy in the source code due to their size - I didn't want to clutter things in the code.

also I'm not sure there's enough need at the time to warrant adding in a factory system here, as the payloads are all unique and will likely continue to be mostly unique, meaning that we would probably end up with more logic in the test suite to load in the mock data than we would have in actual testing code